tests/manifests/e2e/special-cases/no-strategy/controller-config.yaml

```diff
@@ -10,13 +10,4 @@ data:
     group=,kind=ConfigMap;group=,kind=ServiceAccount;group=,kind=Secret;group=,kind=Service;\
     group=rbac.authorization.k8s.io,kind=RoleBinding;group=rbac.authorization.k8s.io,kind=Role"
     defaultUpgradeStrategy: "no-strategy"
-    progressive:
-      defaultAssessmentSchedule:
-        - kind: Pipeline
-          schedule: "120,360,60,10"
-        - kind: MonoVertex
-          schedule: "120,360,60,10"
-        - kind: InterstepBufferService
-          schedule: "0,0,0,10"
-      analysisRunTimeout: "1200"
```

Comment:
I noticed that for the ppnd ConfigMap we're not setting this at all, so for parity I'm making this one the same.